blog/settings.py

```diff
@@ -2,7 +2,7 @@
 
 import environ
 
-env = environ.Env(DEBUG=(bool, False),)
+env = environ.Env(DEBUG=(bool, True),)
```

Comment:
Could I do this when merging? In master branch it's ok. Just for minimize count of commits